src/String.ts

```diff
@@ -95,6 +95,28 @@ Object.defineProperties(String.prototype, {
 			return BigInt(this);
 		},
 	},
+	equalsIgnoreCase: {
+		enumerable: false,
+		configurable: true,
+		writable: true,
+		value: function(compareString: String){
+			return this.toLowerCase() === compareString.toLowerCase();
+		}
+	},
+	count: {
```

Comment:
you are checking letter for only a single character, however letter could be also a string